zbuild.zon

```diff
@@ -117,6 +118,7 @@
                     .blst,
                 },
             },
+            .filters = .{},
```

Comment:
Best to create an GitHub issue since the filtering mechanism is not yet implemented.